From 36dcbfae2b73bc541a0f94780b28c3d5e2cdb1e7 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Sun, 5 Mar 2023 13:01:35 +0900
Subject: [PATCH] Revert "hwdb: fix swapped buttons for Logitech Lift left"

This reverts commit 81cfea95e51c72da6765b517e7038b3e7e3dec9f.

The modalias seems to match a generic Logitech USB receiver even the
connected mouce is not for left hand.

Fixes #26671 and #26676.

(cherry picked from commit ae9f36397f45dff8e2181c6af5f763e51a7c9639)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 6ab1468052..10dde4dbec 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -1328,11 +1328,6 @@ evdev:input:b0003v046DpC309*
  KEYBOARD_KEY_c01b6=images                              # My Pictures (F11)
  KEYBOARD_KEY_c01b7=audio                               # My Music (F12)
 
-# Logitech USB Receiver Mouse (Lift for Business, left-handed)
-# This left-handed ergonomic mouse swaps the left/right buttons
-evdev:input:b0003v046DpC548e0111*
- KEYBOARD_KEY_90001=btn_right
- KEYBOARD_KEY_90002=btn_left
 
 ###########################################################
 # Maxdata
